Backends/CLX/port.lisp

```diff
@@ -193,7 +193,8 @@
                                        event-mask))))
       (port-register-mirror (port sheet) sheet window)
       (when map
-        (xlib:map-window window))))
+        (xlib:map-window window)
```

Comment:
Let me know if you find this kind of feedback annoying. I think the commit message would be more correct like this:

> This issue is not visible with frames because they do draw on the medium *which* flushes clx's buffer. When we only adopt a standalone sheet to a graft *the* problem becomes apparent - *the* window is not mapped until *the* next xlib operation *causes* xlib *to* send buffered requests.

Reply:
fixed